src/webgl/p5.RendererGL.js

```diff
@@ -1083,6 +1086,10 @@ p5.RendererGL.prototype._setFillUniforms = function(fillShader) {
   if (this._tex) {
     fillShader.setUniform('uSampler', this._tex);
   }
+  if (this._tint) {
+    fillShader.setUniform('uTint', this._tint);
+  }
```

Comment:
This causes this example sketch to break:
```
let img;
function preload() {
  img = loadImage('../webgl/assets/UV_Grid_Sm.jpg');
}
function setup() {
  createCanvas(400, 200, WEBGL);
  image(img, 0, 0, 100, 100);
  tint(0, 153, 204); // Tint blue
  image(img, 100, 0, 100, 100);
}
```
The `uTint` uniform in the shader will have a value of vec4(0, 0, 0, 0) when drawing the first image. There are a number of different solutions that will work. You can initialize `_tint` with a value in RendererGL or add an else to this conditional that send a default value when `_tint` is null.

Reply:
i would recommend using a default value instead of a conditional.

Reply:
I have added default value for tint and a couple of unit tests. I wanted to add a unit test for testing whether ```_tint``` is reset after ```draw()``` loop. How can do that?

Reply:
I think you need to use a Promise that tests _tint on the second pass through `draw` before calling `resolve` or `reject`. There may be better examples elsewhere but one place to look is [the structure.js tests](https://github.com/processing/p5.js/blob/master/test/unit/core/structure.js).